quick-start/examples/n6.min.js

```diff
@@ -0,0 +1,24 @@
+import { Scene, Circle, Rectangle } from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: "#45a8ff",
+});
+const { width, height } = scene;
+// Initialise the target position
+scene.cursorPosition.set(scene.center).subtract(0, height / 4);
+
+const sun = new Circle(scene.center.add(0, height / 2), 100, {
+    fill: "orange",
+});
+
+const ocean = new Rectangle([0, height / 2], width, height / 2, {
+    fill: "#28488f",
+});
+
+// Listen to the "draw" event fired each frame (about 60 times a second)
+scene
+    .add(sun, ocean)
+    .startLoop()
+    .on("draw", () => {
+        sun.position.lerp(scene.cursorPosition, 0.05);
```

Comment:
As an experienced developer I have never heard of `lerp` and it does not seem otherwise explained. What does it do? I don't get it from the comment.

Update: I saw it explained in the HTML files. If those texts are nearby, maybe this text doesn't need an explanation.

Reply:
They are kind of nearby in the resulting page. I'll add a little comment to better hammer it down.